server.js

```diff
@@ -551,10 +551,16 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
       var buildType = match[3];  // eg, `ru.yandex.qatools.allure:allure-core:master`.
       var metric = match[4];
       var format = match[5];
-      var realMetric = metric === 'tech_debt' ? 'sqale_debt_ratio' : metric
+      var sonarMetric = metric;
+      if (metric === 'tech_debt') {
+        sonarMetric = 'sqale_debt_ratio';
+      } else if (metric === 'lines') {
+        sonarMetric = 'ncloc';
```

Comment:
The else is unnecessary since it sets sonarMetric = metric by default, but I can change it to use an else if you prefer that.  Sonar has a "lines" metrics and a "ncloc" metric, the former being total lines and the latter being lines of code.  I would guess most people would be interested in lines of code, but maybe not.